scipy/optimize/tests/test_brent.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import division
+
+from numpy.testing import (assert_almost_equal, assert_array_less, assert_)
+
+from scipy.optimize._brent import find_minimum
+
+def test_strictly_convex():
+    def func(x, s):
+        return (x - s)**2 - 0.8
+
+    (x, fx, niters) = find_minimum(lambda x: func(x, 0), -10, 10)
+    assert_almost_equal(x, 0)
+    assert_almost_equal(fx, -0.8)
+    assert_array_less(niters, 7)
```

Comment:
Can you tighten this test? The quadratic fitting should ensure that for an actual quadratic, you jump straight to the minimum. Is there a way to test this property? I suppose there will be a certain number of extra evaluations required for the minimizer to stop exploring.